### ISSUE TITLE
Docs, any preference? 

### ISSUE THREAD

**Issue by koaning** [pola-rs/valve#2]
@ritchie46 Do we have any preference for documentation? I could go for something like `mkdocs` but I figured checking in first because it wouldn't fit the current documentation style.

**Comment by ritchie46** [comment]
Yeah, I've tried a lot. Eventually I concluded that sphinx is most versatile, but requires some setting up. We could simply copy paste that from the main polars repo. Then the setup is done. :)

Once consistent code style and documentation practice keeps mental overhead lower.
